### DISCUSSION TITLE
DnD Character Sheet by @Kylomax

### DISCUSSION THREAD

**Discussion by sigrunixia** [Obsidian-TTRPG-Community/fantasy-statblocks#169] [Show and tell]
This thread is for @kylomax and their DnD Character Sheet.

---

[5e Character Sheet.json.zip](https://github.com/valentine195/obsidian-5e-statblocks/files/9679167/5e.Character.Sheet.json.zip)
I made a character sheet for Dungeons & Dragons 5e. It doesn't look as good as the D&D Beyond character sheet, but it's convenient to have it directly in Obsidian. The skills table is far too many values for one table. If vertical tables were possible in this plugin, it would look far better. Here are some screenshots (The character is a generic character used for testing):
<img width="831" alt="screen shot of character sheet 1" src="https://user-images.githubusercontent.com/77210114/193145169-e7a44dba-fe9b-489c-b782-cf51543265b5.png">
<img width="826" alt="screen shot of character sheet 2" src="https://user-images.githubusercontent.com/77210114/193145183-223d6027-f8a6-4ebd-937b-732140bcd911.png">

### Follow-Up
https://github.com/valentine195/fantasy-statblocks/discussions/41#discussioncomment-3766708 Javalent asked:
What would a vertical table look like in this case?

https://github.com/valentine195/fantasy-statblocks/discussions/41#discussioncomment-3766732 @kylomax replied:
In my opinion, something like:

Table header 1
table value 1
Table header 2
table value 2

would work well. The values would be centered beneath the headers, though.

https://github.com/valentine195/fantasy-statblocks/discussions/41#discussioncomment-5237188 - @Hapadbeep24 Had a Query:
Love this character sheet. Probably the first I've seen for obsidian. Is there a way I can get the empty statblock that I can use to make multiple characters? I don't know if there is a way to do this in the plugin, but if there is and you can direct me to that would be perfect too!

---

**Original Thread**: https://github.com/valentine195/fantasy-statblocks/discussions/41#discussioncomment-3766693

**Comment by eurghwhy** [comment]
@kylomax any chance of getting the example markdown for this?

**Reply by kylomax** [reply]
Hey! Sorry I missed the previous request for it. I have attached it as a yaml. I used it as a homebrew monster

[Example character sheet.txt](https://github.com/javalent/fantasy-statblocks/files/13433773/Example.character.sheet.txt)
